### DISCUSSION TITLE
How uninstall?

### DISCUSSION THREAD

**Discussion by HOBA3BE3DA** [voicepaw/so-vits-svc-fork#589] [General]
How uninstall? I I downloaded the neural network, I didn't like it. The neural network folder weighs 5 GB and during installation I spent 20 GB.

**Comment by Grownz** [comment]
That depends on the system you installed it, i guess.
I use Anaconda on Windows-PC, so the way to go would be:
`conda remove --name myenv --all` with "myenv" being the name of your specific environment.

You can read about that in the official Conda Docs:
https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#removing-an-environment
